### PR TITLE
Removes readme and contributing.md from generate heading lint

### PR DESCRIPTION
**Pull request by harish-sethuraman**
Removes readme and contributing.md from generate heading

### CODE REVIEW

File: beta/package.json [hunk 1]
```diff
@@ -100,6 +100,6 @@
   },
   "lint-staged": {
     "*.{js,ts,jsx,tsx,css}": "yarn prettier",
-    "*.md": "yarn fix-headings"
+    "!(*README|CONTRIBUTING).md": "yarn fix-headings"
```
Comment (gaearon) [inline beta/package.json:103]: Why * before README?

Comment (gaearon) [reply beta/package.json:103]: Can we just scope it down to src/**/*.md?

Comment (harish-sethuraman) [reply beta/package.json:103]: We can do that :)

**Review by gaearon — APPROVED** [review]
have you verified the commit hook works as expected?

**Comment by harish-sethuraman** [comment]
Yep I did

**Pull request merged**